epaint/src/tessellator.rs

```diff
@@ -286,6 +286,12 @@ pub struct TessellationOptions {
 
     /// If true, no clipping will be done.
     pub debug_ignore_clip_rects: bool,
+
+    /// The maximum distance between the original curve and the flattened curve.
+    pub bezier_flattern_tolerence: f32,
```

Comment:
There was a typo, but I also think this is a sufficient name:

```suggestion
    pub bezier_tolerence: f32,
```

Reply:
fixed in PR #1178